controller/share_manager_controller.go

```diff
@@ -798,6 +860,33 @@ func (c *ShareManagerController) createShareManagerPod(sm *longhorn.ShareManager
 		return nil, err
 	}
 
+	var affinity *corev1.Affinity
+
+	if pv.Spec.StorageClassName != "" {
+		sc, err := c.ds.GetStorageClass(pv.Spec.StorageClassName)
+		if err != nil {
+			c.logger.WithError(err).Warnf("Failed to get storage class %v", pv.Spec.StorageClassName)
```

Comment:
NIT: "..., will continue the share manager pod creation"

Reply:
Updated. Thank you.